corehq/apps/app_manager/suite_xml/contributors/scheduler.py

```diff
@@ -0,0 +1,32 @@
+from corehq.apps.app_manager import id_strings
+from corehq.apps.app_manager.suite_xml.basic import SuiteContributor
+from corehq.apps.app_manager.suite_xml.xml import Schedule, ScheduleFixture, ScheduleVisit
+
+
+class SchedulerContributor(SuiteContributor):
+    section = 'fixtures'
+    
+    def contribute(self):
+        self.suite.fixtures.extend(self.fixtures)
+
+    @property
+    def fixtures(self):
```

Comment:
I'm assuming there is a way we can also add other sections here (e.g. details)? That would be ideal, so that all the schedule stuff lives in one file. 
Otherwise, this might get a bit confusing if there is some schedule related stuff scattered around the other section contributor classes.